<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DATOS DEL LIBRO A EDITAR:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>titulo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_titulo_libro">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>paginas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_paginas_libro">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>precio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entrada_precio_libro">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_guardar_cambios_libro">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>490</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>GUARDAR CAMBIOS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_digital">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string> version digital disponible</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_tapa">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>tapa blanda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tapa dura</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tapa:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>opcion de envio:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_standar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Envio Standar</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_urgente">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Envio Urgente</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_prioritario">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Envio Prioritario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Portada:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_seleccionar_archivo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Seleccionar Archivo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagen">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_paginas">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>error paginas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error_titulo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>error titulo</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
